test/unittests/mongoBackend/mongoGetSubscriptions_test.cpp

```diff
@@ -96,12 +96,12 @@ static void prepareDatabaseV1Subs(void) {
                                                        "type" << "ONCHANGE" <<
                                                        "value" << BSON_ARRAY("AX2" << "AY2")
                                                        )) <<
-                        "throttling" << 5.0
+                        "throttling" << (long long) 5
                         );
 
     BSONObj sub3 = BSON("_id" << OID(SUB_OID3) <<
-                        "expiration" << 20000000 <<
-                        "lastNotification" << 25000000 <<
+                        "expiration" << (long long) 20000000 <<
```

Comment:
I don't know if that is the way to go.
Very unfortunate this whole int/long thing ...
All these checks make us lose time.
If we could avoid them, it would be great.

However, we have an issue for this, the #1455